From b5abd3f9664c396ad57f177750973695c58e8b7f Mon Sep 17 00:00:00 2001
From: Michael Gratton <mike@vee.net>
Date: Mon, 28 Sep 2020 08:53:35 +1000
Subject: [PATCH 031/124] build: Fix build failure due to missing client API

Don't use client lib vala ags when building the web process extension,
since that will cause it to also write a VAPI with the same name as
the client lib, causing the build to fail if the web process side wins
that race.

Fixes #985
---
 src/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index 87760e74..225777e8 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -72,7 +72,7 @@ web_process = library('geary-web-process',
     webkit2gtk_web_extension,
   ],
   include_directories: config_h_dir,
-  vala_args: client_vala_args,
+  vala_args: geary_vala_args,
   c_args: geary_c_args,
   install: true,
   install_dir: web_extensions_dir
-- 
2.29.2

